index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+			rel="stylesheet"
+		/>
+		<link
+			rel="stylesheet"
+			href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+		/>
+		<link rel="stylesheet" href="style.css" />
+		<title>youtube clone</title>
+	</head>
+	<body>
+		<h1>changed</h1>
+		<nav class="navigation--menu">
+			<div class="hamburger--menu">
+				<img src="img/download.png" alt="a videoHamburger" />
```

Comment:
Your directory name is not matching here, `img/download.png` with `Img/download.png`